examples/online-store/step-definitions/custom/IngestOrders/IngestOrders.step.json

```diff
@@ -1,9 +1,25 @@
 {
+  "language" : "zxx",
   "name" : "IngestOrders",
+  "description" : null,
   "type" : "CUSTOM",
```

Comment:
Changing the type to "ingestion" here and in the flow might resolve the issue with LoadOrders not working through the QuickStart UI.

Reply:
Yeah, this should cause a pull to happen when ran. If its of type ingest, it'll 'pull' when 'ran' from DHF.

Reply:
@ryanjdew Will I also need to define the files to ingest? Currently those are defined in the MlcpTask in build.gradle. I'm not clear on how to define a flow and its files to load that will work via both QS and MlcpTask.

Reply:
@rjrudin The MLCP task will ignore any file input information in the step/step definition. To the step in the flow, you can information that looks like this, so it will run in QuickStart (fingers crossed) https://github.com/marklogic/marklogic-data-hub/blob/develop/marklogic-data-hub/src/test/resources/master-test/flows/myNewFlow.flow.json#L11